Source/RequestBuilder.swift

```diff
@@ -465,11 +465,43 @@ class RequestBuilder: NSObject {
         }
 
         requestUserExt["data"] = Targeting.shared.getUserDataDictionary().getCopyWhereValueIsArray()
+        
+        if let externalUserIdArray = getExternalUserIds(), externalUserIdArray.count != 0{
+            requestUserExt["eids"] = externalUserIdArray
+        }
```

Comment:
I think that the next code snippet will have the same effect because of `getObjectWithoutEmptyValues()`. But it should be checked 

``` Swift
// no extra checks 
requestUserExt["eids"] = getExternalUserIds()
```

Reply:
Yes, updated the same